### PR TITLE
Add subscription of BC.OnSystemCapabilityUpdated and AS.OnAppServiceData notifications sent from SDL

### LINKED ISSUE #257: HMI doesn't show in console log BC.OnSystemCapabilityUpdated and AS.OnAppServiceData notifications sent from SDL
According to HMI API `BC.OnSystemCapabilityUpdated` and `AS.OnAppServiceData` notifications are bi-directional and can be sent from/or to SDL.
Currently HMI doesn't show in console notifications which are sent from SDL.
Seeing these messages will be very helpful during manual testing of App Services

### PR DESCRIPTION
**Pull request by dboltovskyi**
Implements/Fixes #257  

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
Add subscription for `BC.OnSystemCapabilityUpdated` and `AS.OnAppServiceData` notifications sent from SDL

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: ffw/BasicCommunicationRPC.js [hunk 1]
```diff
@@ -58,6 +58,8 @@ FFW.BasicCommunication = FFW.RPCObserver
       onSDLConsentNeededSubscribeRequestID: -1,
       onResumeAudioSourceSubscribeRequestID: -1,
       onServiceUpdateNotificationSubscribeRequestID: -1,
+      onSystemCapabilityUpdatedNotificationSubscribeRequestID: -1,
+      //
```
Comment (AKalinich-Luxoft) [inline ffw/BasicCommunicationRPC.js]: @dboltovskyi please remove this

File: ffw/BasicCommunicationRPC.js [hunk 2]
```diff
@@ -145,6 +149,8 @@ FFW.BasicCommunication = FFW.RPCObserver
           .subscribeToNotification(this.onResumeAudioSourceNotification);
         this.onServiceUpdateNotificationSubscribeRequestID = this
           .subscribeToNotification(this.onServiceUpdateNotification);
+        this.onSystemCapabilityUpdatedNotificationSubscribeRequestID = this
```
Comment (AKalinich-Luxoft) [inline ffw/BasicCommunicationRPC.js]: @dboltovskyi its just a subscription. Is handler for this notification already exists?

File: ffw/BasicCommunicationRPC.js (hunk 1 above)
Comment (dboltovskyi) [reply ffw/BasicCommunicationRPC.js]: @AKalinich-Luxoft Fixed in 43f4d69

File: ffw/BasicCommunicationRPC.js (hunk 2 above)
Comment (dboltovskyi) [reply ffw/BasicCommunicationRPC.js]: @AKalinich-Luxoft It's not expected actions from HMI on these notifications

**Pull request merged**